integration/tilt.go

```diff
@@ -37,6 +49,16 @@ func (d *TiltDriver) cmd(args []string, out io.Writer) *exec.Cmd {
 	for k, v := range d.Environ {
 		cmd.Env = append(cmd.Env, fmt.Sprintf("%s=%s", k, v))
 	}
+	hasPortArg := false
+	for _, arg := range args {
+		if strings.HasPrefix("--port=", arg) {
+			hasPortArg = true
+			break
+		}
+	}
+	if !hasPortArg && d.Port > 0 {
+		cmd.Env = append(cmd.Env, fmt.Sprintf("TILT_PORT=%d", d.Port))
```

Comment:
mostly out of curiosity, is there a reason you're using the env variable instead of the port flag here? would rather be consistent within the test, since it seems likely this will bite someone later.

Reply:
The goal here was to have some sane default behavior but get out of the way if you explicitly want to do something with ports.

Since `--port` isn't a global flag, it could make some Tilt commands fail, so this was easier than adding it explicitly to each wrapper, and so the check to omit it if `--port` was defined was to make sure there is an escape hatch, but I can make it a bit more explicit, e.g. have `AutomaticPortSelection bool` on the fixture and don't touch it if false?

Reply:
^ kinda did something similar, made it more explicit and throw hard errors if you're in "automatic port" mode and try to specify either `--port` or `TILT_PORT` so there should be no surprises